packages/swap/src/pricing/index.ts

```diff
@@ -1,58 +1,97 @@
 import axios from 'axios';
 import { InternalAsset } from '@/shared/enums';
 import env from '../config/env';
-import { CacheMap } from '../utils/dataStructures';
```

Comment:
we removed something from the `CacheMap` to make it work for this usecase, so maybe we should readd that?
https://github.com/chainflip-io/chainflip-sdk-monorepo/pull/570/files#diff-b0ae927d6867edde94cb13c8bf6099e9f97f6e27905f178c4e481a70ec814e5dL12

Reply:
done